fv3core/stencils/a2b_ord4.py

```diff
@@ -394,116 +395,151 @@ class AGrid2BGridFourthOrder:
     """
 
     def __init__(
-        self, grid_type, kstart: int = 0, nk: int = None, replace: bool = False
+        self,
+        grid_indexing: GridIndexing,
+        agrid1,
+        agrid2,
+        bgrid1,
+        bgrid2,
+        dxa,
+        dya,
+        edge_n,
+        edge_s,
+        edge_e,
+        edge_w,
+        grid_type,
```

Comment:
Yes, we'll be here ages otherwise. I considered adding a comment for which variables come from the namelist vs the grid, but stopped when I realized that the translate tests document this automatically.